src/implementation/Client/GRPCClient/configuration.ts

```diff
@@ -51,6 +51,7 @@ export default class GRPCClientConfiguration implements IClientConfiguration {
       for (const [key, value] of Object.entries(metadataObj)) {
         metadata.add(key, value);
```

Comment:
We do not need to pass `metadata` anymore to the call since we are populating `msg.getMetadataMap()`.